app/views/kaminari/_gap.html.haml

```diff
@@ -0,0 +1,9 @@
+-#
+  Non-link tag that stands for skipped pages...
+  available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+%span.page.gap
+  = sanitize t('pagination.truncate')
```

Comment:
Why the `sanitize` call?

Reply:
Ah, but `sanitize` allows a some of HTML tags...
Would it be better to call `sanitize t('pagination.truncate'), tags: [], attributes: []` to disallow them?

Reply:
ce59dce